client/js/render.js

```diff
@@ -0,0 +1,192 @@
+"use strict";
```

Comment:
Sure, but if this is something we want to enforce we should probably see if there's an eslint rule (or write our own) so we don't have to point it out.